byoqm/runner/runner.py

```diff
@@ -75,11 +78,13 @@ def _run_metrics(self) -> Dict:
         results = {}
 
         metrics = self._model.getDesc()["metrics"]
-        for metric, exec_path in metrics.items():
-            cmd = [f"./{exec_path}", self._src_root]
-            process = subprocess.run(cmd, stdout=subprocess.PIPE)
-            result = process.stdout.decode("utf-8").strip()
-            results[metric] = int(result)
+        for metric, metric_file in metrics.items():
+            spec = importlib.util.spec_from_file_location("metric", metric_file)
+            module = importlib.util.module_from_spec(spec)
+            sys.modules[metric] = module
+            spec.loader.exec_module(module)
+            module.metric.set_coordinator(self._coordinator)
+            results[metric] = int(module.metric.run())
```

Comment:
This goes against what is happening in the `_run_metrics` method in the same class. Not sure which one is more correct, but don't think we should have the functionality twice.

Reply:
Nevermind, checkout out the code locally, I can see this is in the `_run_metrics` branch.